docs/source/guide/tutorials/base/how_to_train/action_detection.rst

```diff
@@ -0,0 +1,145 @@
+Action Detection  model
+================================
+
+This live example shows how to easily train, validate, optimize and export spatio-temporal action detection model on the subset of `JHMDB <http://jhmdb.is.tue.mpg.de/>`_.  
+
+.. note::
+
+  To learn deeper how to manage training process of the model including additional parameters and its modification, refer to :doc:`./detection`.
+
+The process has been tested on the following configuration.
+
+- Ubuntu 20.04
+- NVIDIA GeForce RTX 3090
+- Intel(R) Core(TM) i9-10980XE
+- CUDA Toolkit 11.1
+
+*************************
+Setup virtual environment
+*************************
+
+You can follow the installation process from a :doc:`quick start guide <../../../get_started/quick_start_guide/installation>` to create a universal virtual environment for OTX.
+
+***************************
+Dataset preparation
+***************************
+
+Although we offer conversion codes from `ava dataset format <https://github.com/open-mmlab/mmaction2/blob/master/tools/data/ava/README.md>`_ to `cvat dataset format <https://opencv.github.io/cvat/docs/manual/advanced/xml_format/>`_ from `this code <../../../../../../otx/algorithms/action/utils/convert_convert_data_to_cvat.py>`_, for easy beginning you can download subset of JHMDB dataset, which already transformed to CVAT format from `this link <https://drive.google.com/file/d/1ZgUYkhOAJ9_-xMCujPJlMLFILuGkhI4X/view?usp=share_link>`_.
+
+If you download data from link and extract to training_extensions/data folder(you should make data folder at first), you can see the structure below.
+
+.. code-block::
+
+    training_extensions
+    └── data
+        └── JHMDB_5%
```

Comment:
How about to change `JHMDB_5percent`? Special character may not be reliable for all situations.